Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'validates_email_format_of'
 gem 'friendly_id', '~> 5.2.1'
 gem 'babosa', '1.0.2'
 gem 'nokogiri', '~> 1.7.2'
-gem 'slimmer', '~> 11.0'
+gem 'slimmer', '~> 11.0.2'
```

Comment:
We don't need to change this. `bundle update slimmer` will pick up the new `x.x.n` version.